### PR TITLE
typo-ish: improved comment that indicates state of Barrier

### PR DESCRIPTION
**Pull request by cpriyank**
removed extra comma in comment that indicates state of a `Barrier` as it was confusing and breaking the flow while reading. Also added missing space after `#` that began the comment

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@cpriyank

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Review by serverwentdown — APPROVED** [review]
LGTM, erroneous extra comma in code comment.

**Review by eamanu — CHANGES_REQUESTED** [review]
I'm not sure about a new commit for only for this change.

We can take in advantage some other bugs to fix this.

Also @cpriyank don't forget sign the CLA.

File: Lib/threading.py [hunk 1]
```diff
@@ -608,7 +608,7 @@ def __init__(self, parties, action=None, timeout=None):
         self._action = action
         self._timeout = timeout
         self._parties = parties
-        self._state = 0 #0 filling, 1, draining, -1 resetting, -2 broken
+        self._state = 0 # 0 filling, 1 draining, -1 resetting, -2 broken
```
Comment (ambv) [inline Lib/threading.py]:
```suggestion
        self._state = 0  # 0 filling, 1 draining, -1 resetting, -2 broken
```

**Comment by ambv** [comment]
Closing in favor of GH-28623.

**Pull request closed without merging**